tutorial/deploy_jaas_microk8s.rst

```diff
@@ -10,74 +10,43 @@ This tutorial will teach you about JIMM and Juju as well as a bit about Canonica
 
 Prerequisites
 -------------
+- A workstation, e.g., a laptop, that has sufficient resources to launch a virtual machine with 4 CPUs, 8 GB RAM, and 50 GB disk space.
 
-For this tutorial you will need the following:
 
-- An Ubuntu machine.
+Set up an isolated test environment
+-----------------------------------
 
-Setup Multipass (Optional)
---------------------------
-Multipass is a tool to launch Ubuntu VMs from Windows, Linux and MacOS. The remainder of this guide can be run from within a Multipass VM to avoid affecting the host machine.
+Set up an isolated test environment with Multipass and the `charm-dev` blueprint, which will provide all the necessary tools and configuration for the tutorial (a localhost machine cloud and Kubernetes cloud, Juju, etc.). 
 
-Start by running the following commands to install and start a Multipass VM, the optional section will define the VM's memory/CPU/disk usage.
+On your machine, install Multipass and use it to set up an Ubuntu virtual machine (VM) called `my-juju-vm` from the `charm-dev` blueprint. 
 
-.. code:: bash
+.. note::
+    See more: `Set things up (automatically) <https://canonical-juju.readthedocs-hosted.com/en/latest/user/howto/manage-your-deployment/manage-your-deployment-environment/#manage-your-deployment-environment>`__  
 
-    sudo snap install multipass
-    multipass launch jammy --name jimm-deploy [-m 12g -c 4 -d 40G]
-    multipass shell jimm-deploy
+    Note: This document also contains a manual path, using which you can set things up without the Multipass VM or the `charm-dev` blueprint. However, please note that the manual path may yield slightly different results that may impact your experience of this tutorial. 
```

Comment:
Should we not make this more evident since in our case things don't yet work without Multipass?